README.md

```diff
@@ -1,29 +1,52 @@
 # Admin Tool for RubiX
 
+**NOTE:** RubiX Admin currently only supports RPM-based Linux distributions.
+
 ## Installation
-`rubix-admin` should be installed on the cluster (Presto/Spark/Hadoop2) master
+
+RubiX Admin should be installed on the Presto/Spark/Hadoop2 cluster master node.
+
+### From Source
 
     git clone https://github.com/qubole/rubix-admin.git
     virtualenv venv
     source venv/bin/activate
     python setup.py install
-    rubix-admin -h
-    
-## Configure Admin Tool
-`rubix-admin` expects a configuration file at `~/.radminrc`. If you followed
-the instruction in the previous section, it should already be present. 
-If the configuration file does not exist, run:
+
+### From PIP
+
+    pip install rubix-admin
+
+## Configuration
+RubiX Admin expects a configuration file at `~/.radminrc`. This file can be
+generated by running the following command:
 
     rubix-admin -h
```

Comment:
i think its` rubix_admin -h` ? @jordanw-bq can you double check

Reply:
You are correct. Updated all command references to `rubix_admin`